package-lock.json

```diff
@@ -541,13 +541,10 @@
       }
```

Comment:
Can you please revert the changes in `package-lock.json`. Are they intentional?

Reply:
@naveedkhan8067 Yes, I can revert.  I got a "severe/critical" warning on npm install and followed the prompt to do "npm audit fix" which eliminated the highest severity vulnerability and some of the moderate ones.  They are intentional in the sense that I was aware of the changes and deliberately staged, committed, and pushed them as I'm accustomed to do in other repos.  LMK and I'll revert promptly.

Reply:
Thanks @jchick-bentley. I will take a look to those warnings.